docs/sql/statements/begin.rst

```diff
@@ -32,8 +32,9 @@ Description
 
 The statement starts a transaction block until it committed or rolled back.
 
-As CrateDB does not support transactions, this command has no effect and will
-be ignored.
+As CrateDB does not support transactions, the only effect of this command is
+to start a scope on which cursors ``WITHOUT HOLD`` can be
```

Comment:
Shouldn't this be `WITH HOLD`?

Reply:
No, I found it confusing as well, but it is the cursors `WITHOUT HOLD` that require the "transaction" context.
This is the same in other systems, "WITH HOLD specifies that the cursor can continue to be used after the transaction that created it successfully commits" ( https://www.postgresql.org/docs/current/sql-declare.html )